public/documentation/embedding_crosswalk/crosswalk_aar.md

```diff
@@ -114,6 +114,8 @@ Then install it into the local Maven repository:
     }
     ```
 
+    Using mavenLocal() instead of remote repo url to refer local aar.
+
```

Comment:
I don't understand this line in the context of the article.  Are you saying that we are "Using mavenLocal()" instead of the remote repo url?  I didn't see "mavenLocal" used anywhere.   Or are you telling the developer to "Use mavenLocal()..." ?